### PR TITLE
Enabling Bazelisk to be used when Bazel is not present during install build from source

### PR DESCRIPTION
**Pull request by Gelesh**
Update Configure.py @55218, Last commit failed at CICD for seemly unrelated issue. This patch is just to re run the same CICD and validate

### CODE REVIEW

**Comment by mihaimaruseac** [comment]
Please use a better PR title. See https://cbea.ms/git-commit/

**Comment by Gelesh** [comment]
> Please use a better PR title. See https://cbea.ms/git-commit/

Do I need to make a fresh PR ? Could you please review this ?

**Comment by mihaimaruseac** [comment]
You can update the title of this PR

**Comment by Gelesh** [comment]
> You can update the title of this PR

Done, hope it looks good. Thanks for your feedback

File: configure.py [hunk 1]
```diff
@@ -477,8 +477,12 @@ def check_bazel_version(min_version, max_version):
   """
   bazel_executable = which('bazel')
   if bazel_executable is None:
-    print('Cannot find bazel. Please install bazel.')
-    sys.exit(1)
+    bazel_executable = which('bazelisk')
+    if bazel_executable is None:
+      print('Cannot find bazel. Please install bazel.')
```
Comment (mihaimaruseac) [inline configure.py:482]:
```suggestion
      print('Cannot find bazel. Please install bazel/bazelisk.')
```

File: configure.py [hunk 2]
```diff
@@ -477,8 +477,12 @@ def check_bazel_version(min_version, max_version):
...
+      sys.exit(1)
+    else:
+      print('Using bazelisk, since bazel not found')
```
Comment (mihaimaruseac) [inline configure.py:485]: Let's not add additional log messages.

File: configure.py [hunk 3]
```diff
@@ -477,8 +477,12 @@ def check_bazel_version(min_version, max_version):
...
   bazel_executable = which('bazel')
   if bazel_executable is None:
-    print('Cannot find bazel. Please install bazel.')
-    sys.exit(1)
+    bazel_executable = which('bazelisk')
+    if bazel_executable is None:
+      print('Cannot find bazel. Please install bazel/bazelisk.')
+      sys.exit(1)
+    else:
+      print('Using bazelisk, since bazel not found')
```
Comment (mihaimaruseac) [inline configure.py:485]:
```suggestion
```

**Review by mihaimaruseac — APPROVED** [review]

File: configure.py (hunk 2 above)
Comment (Gelesh) [reply configure.py]: sure

**Comment by Gelesh** [comment]
@mihaimaruseac , any thought on the internal CI fail Internal

**Pull request merged**